awx/main/models/projects.py

```diff
@@ -185,11 +185,11 @@ def clean_credential(self):
 
     def resolve_execution_environment(self):
         """
-        Project updates, themselves, will use the default execution environment.
+        Project updates, themselves, will use the control plane execution environment.
         Jobs using the project can use the default_environment, but the project updates
         are not flexible enough to allow customizing the image they use.
         """
-        return get_default_execution_environment()
+        return get_control_plane_execution_environment()
```

Comment:
This makes it so project updates run with the "control plane EE".

Should ec2 inventory updates also run with the "control plane EE"? Why or why not?

Reply:
I don't have a strong opinion about it, but presumably there was a reason we hadn't had that in place previously.  @shanemcd ?

Reply:
- Inventory updates don't run on the control plane. 
- Users are able to choose which EE they want to use at the inventory source level.